diffpy/pdfmorph/morphs/morphpdftordf.py

```diff
@@ -24,7 +24,7 @@
 class MorphXtalPDFtoRDF(Morph):
```

Comment:
this is not a "morph" this is simply a transformation.  Please can we change this language?  I want to merge this so please make an issue to change the language (and name) on this.

Having said that, leave all the changes you have done and I will merge it,  but let's think a bit about the class name and the docstring/documentation.  I should probably understand exactly what is being done here before we finalize this, but stick it on an issue (and add it to the release milestone) so we loop back to it

Reply:
Yep, I'll open a new issue for this (and below)

Reply:
I am having trouble adding the issue to the milestone